backend-api/src/functions/jwks/jwksBuilder/jwksBuilder.ts

```diff
@@ -35,20 +38,25 @@ export class JwksBuilder implements IJwksBuilder {
     const jwks: Jwks = {
       keys: [],
     };
-    const result = await this.getPublicKeyAsJwk();
-    if (result.isError) {
-      return result;
+
+    for (const keyId of this.keyIds) {
+      const result = await this.getPublicKeyAsJwk(keyId);
+      if (result.isError) {
+        return result;
+      }
```

Comment:
nitpick: this is typically how we name the consts that use the Result pattern

```suggestion
      const getPublicKeyAsJwkResult = await this.getPublicKeyAsJwk(keyId);
      if (getPublicKeyAsJwkResult.isError) {
        return getPublicKeyAsJwkResult;
      }
```